src/format/table.rs

```diff
@@ -167,19 +176,11 @@ impl TableView {
             if max_per_column[head] > max_naive_column_width {
                 headers[head] = fill(&headers[head], max_column_width);
                 for row in 0..entries.len() {
-                    entries[row][head] = fill(&entries[row][head], max_column_width);
+                    entries[row][head].0 = fill(&entries[row][head].0, max_column_width);
                 }
             }
         }
 
-        // Paint the number column, if it exists
-        if entries.len() > 1 {
-            for row in 0..entries.len() {
-                entries[row][0] =
-                    format!("{}", Color::Black.bold().paint(entries[row][0].to_string()));
-            }
-        }
-
```

Comment:
I haven't tested this approach, but one issue I was having if you paint it early is that if this column gets wrapped for being too long, the ANSI sequence might break over two lines and mess up the color.  Do you know if that happens using the row formatting?

(I'm missing where the coloring happens now, too. I might have just overlooked it)

Reply:
I would hope that `prettytable` handles this properly, but I can try to check. The colouring is added on line 63 now, it's tied to the index cell data.

Reply:
I validated that `prettytable` handles colouring correctly; here I coloured some cells red and you can see it did not affect the column separators when wrapped:
![image](https://user-images.githubusercontent.com/12575/63658504-37e73d00-c7ff-11e9-89e2-048fd8bff841.png)